radio/src/storage/eeprom_conversions.cpp

```diff
@@ -1262,6 +1262,11 @@ bool eeConvert()
 
   ALERT(STR_STORAGE_WARNING, msg, AU_BAD_RADIODATA);
 
+// Only do a backup if supported and if it is getting a unique filename
+#if defined(EEPROM_RLC) && defined(RTCLOCK)

```

Comment:
Yes, but currently eeprombackup is only defined if EEPROM_RLC is defined. Also the 480x272 radio_version.cpp lacks the backup functionality. I think backup might not have been implemented for Horus yet.